src/Hosting/Hosting/src/Internal/Host.cs

```diff
@@ -39,19 +39,24 @@ public async Task StartAsync(CancellationToken cancellationToken = default)
         {
             _logger.Starting();
 
-            await _hostLifetime.WaitForStartAsync(cancellationToken);
+            using (var combinedCancellationTokenSource = CancellationTokenSource.CreateLinkedTokenSource(cancellationToken, _applicationLifetime.ApplicationStopping))
```

Comment:
Can we use C#8 in Extensions?
```suggestion
            using var combinedCancellationTokenSource = CancellationTokenSource.CreateLinkedTokenSource(cancellationToken, _applicationLifetime.ApplicationStopping);
```
Less `{ }`

Reply:
I had that in a draft and then opted for a normal using block… 😅 

I don’t have a preference for either though, so I can change that back.